src/matOptimize/Profitable_Moves_Enumerators/Profitable_Moves_Enumerators.hpp

```diff
@@ -6,6 +6,11 @@
 #include <vector>
 #include <array>
 #include "../stack_allocator.hpp"
+struct Move_Found_Callback{
+    bool operator()(Profitable_Moves move,int best_score_change){
+        return move.score_change <= best_score_change;
```

Comment:
This functor is called for every move. Currently, a move is considered as long as it improves the parsimony score. Larch may be able to do more filtering.